app/gui/src/dashboard/components/MarkdownViewer/MarkdownViewer.tsx

```diff
@@ -31,16 +31,23 @@ export function MarkdownViewer(props: MarkdownViewerProps) {
   const logger = useLogger()
   const { getText } = useText()
 
-  const transformImageUrl: UrlTransformer = (path: string) =>
-    /^https?:/.test(path) ?
-      Promise.resolve(Ok({ url: path }))
-    : imgUrlResolver(path).then(
+  const transformImageUrl: UrlTransformer = (path: string) => {
+    // In Enso Documentation, the relative paths are from module's directory
+    // Here we always display docs from `src/Main.enso` module
+    const appliedUrl = new URL(path, 'file:///src')
+    if (appliedUrl.protocol !== 'file:') {
+      return Promise.resolve(Ok({ url: path }))
```

Comment:
Do we really want to allow any non-`file` protocol here? I think users will not need anything other than `https`/`http`, and it seems safer not to hotlink images over arbitrary protocols from user-provided content.

Reply:
We used to use enso:// protocol for fetching internal files from the project. Under the hood we check that the requested path is in a subfolder of the project (for security reasons)

I think we can use the same approach here.

Reply:
I just copied what is in https://github.com/enso-org/enso/blob/develop/app/gui/src/project-view/components/DocumentationEditor/images.ts#L68

Reply:
> I just copied what is in https://github.com/enso-org/enso/blob/develop/app/gui/src/project-view/components/DocumentationEditor/images.ts#L68

Ok, we should change it there. It would be nice if we could share one implementation of the URL normalization logic, that should probably have a protocol whitelist.